cephfs/cephfs_test.go

```diff
@@ -139,26 +209,29 @@ func TestChmodDir(t *testing.T) {
 	assert.NoError(t, err)
 
 	// os.Stat the actual mounted location to verify Makedir/RemoveDir
-	stats, err := os.Stat(CephMountTest + dirname)
+	stats, err := os.Stat(path.Join(CephMountDir, dirname))
 	require.NoError(t, err)
 
 	assert.Equal(t, uint32(stats.Mode().Perm()), stats_before)
 
 	err = mount.Chmod(dirname, stats_after)
 	assert.NoError(t, err)
 
-	stats, err = os.Stat(CephMountTest + dirname)
+	stats, err = os.Stat(path.Join(CephMountDir, dirname))
```

Comment:
`assert.NoError(t, err)` missing?

Reply:
Yep. Looks like it. To be fair to myself it was already missing and I just didn't notice but I'll add it in since the point of the PR is to improve test robustness anyway. :-)